### DISCUSSION TITLE
New Features - Pinterest * Instagram

### DISCUSSION THREAD

**Discussion by Linktydraa** [TumblThreeApp/TumblThree#438] [Ideas]
Hi Thomas,

I was just curious since you've been able to implement a feature to crawl Twitter blogs, do you think you'll ever create a feature to do the same for Instagram and Pinterest pages?

**Comment by thomas694** [comment]
Hello,
at the moment there are no plans for it.
But if someone else likes to do it, we'll gladly embrace it.

**Reply by Linktydraa** [reply]
How come there isn't a current plant to implement that feature? Didn't you implement of gifycat feature?

**Reply by thomas694** [reply]
In order to download Tumblr posts containing GIFs hosted on Gfycat, support for it was implemented a few years ago. It was one of the popular image hosters at that time.

They announce that Gfycat is shutting down as of Sept. 1.

**Comment by pinvid898** [comment]
Yes, there is much potential in both Instagram and Pinterest as well as other social media sites. And you can't believe that nowadays if you want to grow your online business then you must use one of them. Here is a blog that clears your mind about [Instagram and Pinterest](https://pinvid.net/pinterest-vs-instagram/).
